clients/go/resources.go

```diff
@@ -0,0 +1,269 @@
+// Package myanmartools Code generated by go-bindata. (@generated) DO NOT EDIT.
```

Comment:
How is this file generated?  Is there some command you need to run?

Reply:
@sffc No, in the Makefile I wrote the script to run on build `go generate`

Reply:
Do you need this in source control for `go get` of myanmartools to work?

If not, consider removing this file from source control (and adding it to gitignore) since it is auto-generated.

Reply:
@sffc yes I need it for the module to work since, there's no way we can bundle in the asset file in go, we need to bundle `zawgyiUnicodeModel.dat` and `compatibility.tsv` files into this file and read the data through the api provided by the generated package.

Reply:
@sffc currently in golang if a package is meant to be imported by other parties and it also required generated code files to work, the only for that package to be `go get` - able, the only approach is the commit the generated code file required.

Reply:
OK, sgtm